File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/commands/SeatAssignments/AssignSeat.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.SeatAssignments
{
    public class AssignSeat : Command<Guid>
    {
        public int Position { get; set; }
        public PersonalInfo PersonalInfo { get; set; }

        public AssignSeat() { }
        public AssignSeat(Guid orderId) : base(orderId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/seatassignments/AssignSeat.java
package com.microsoft.conference.common.registration.commands.seatassignments;

import com.microsoft.conference.common.registration.commands.PersonalInfo;
import org.enodeframework.commanding.Command;

public class AssignSeat extends Command<String> {
    public int position;
    public PersonalInfo personalInfo;

    public AssignSeat() {
    }

    public AssignSeat(String orderId) {
        super(orderId);
    }
}
